### PR TITLE
[liblsl] Update library to 1.13.0-b11

### PR DESCRIPTION
**Pull request by ehsan-mohammadi**
- Update `liblsl` library to 1.13.0-b11 version.
- Add `fix-install.patch` patch file to remove `.exe` files and avoid build failing.

### CODE REVIEW

File: ports/liblsl/portfile.cmake [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+set(VCPKG_LIBRARY_LINKAGE dynamic)
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO sccn/liblsl
-    REF 1.13.0-b6 # NOTE: when updating version, also change it in the parameter to vcpkg_configure_cmake
-    SHA512 fb98cdd73de5f13e97f639ba3f2f836d46ce28cdcb2246584728f296eb647e2c9c069534470a603b10d7dc34ab8978bf246e2187428ab231a925feb0b7024c89
+    REF 1.13.0-b11
```
[the remark this answers is not in the document]
Comment (ehsan-mohammadi) [reply ports/liblsl/portfile.cmake]: @cbezault I'm so sorry for my mistake. I resolved this conversation.

**Comment by cenit** [comment]
> Add fix-install.patch patch file to remove .exe files and avoid build failing.

@cbezault @ehsan-mohammadi lslver is an executable. Why has it been arbitrarily transformed into a static library and why the change has been approved with a patch?

**Comment by ehsan-mohammadi** [comment]
@cenit Thanks for reviewing this PR! When we try to install `liblsl`, the `lslver.exe` be made and it avoid installing `liblsl` successfully. Do you have any suggestion to solve this problem?

**Comment by cenit** [comment]
liblsl.exe should have been moved to the tools folder or removed, not transformed into a "fake" library.
(Luckily) There is plenty of people now here in this repo marked as "internal" in vcpkg but still some basic checks when accepting PRs are still missing from the table. It's not a race to "do work", it should be a race to do "good work".

**Comment by ehsan-mohammadi** [comment]
@cenit I know, and thanks a lot for helping me. I'll fix this issue.

**Comment by cenit** [comment]
no problem, it's not against you of course.
I like your good attitude and the fact that you're trying your best, and doing a lot of work. I can't find a common topic in your PRs so (guessing) maybe you just want to contribute to this project without much in return, and this is wonderful. 
Telling you to improve the "product" (the PR) should be others' work, in order to make your efforts even more fruitful, so I am also sorry if you see me rude.

On your side, since it seems that the green mark is usually enough to grant a merge, please try to do your best to improve port and not just modify references to a newer release: check every patch (new and already existing) to be sure that works and is valid, check tools if properly installed with a feature, verify if any feature is missing or non-working (CI, for ports without dependencies that might trigger extra ones, just checks default features, unfortunately)

**Comment by ehsan-mohammadi** [comment]
@cenit Thanks a lot. I will warmly welcome your comments.

**Pull request merged**